test/test_distributions.py

```diff
@@ -1514,7 +1517,7 @@ def test_entropy_shape(self):
                     message = '{} example {}/{}, shape mismatch. expected {}, actual {}'.format(
                         Dist.__name__, i + 1, len(params), expected_shape, actual_shape)
                     self.assertEqual(actual_shape, expected_shape, message=message)
-                except NotImplementedError:
+                except (NotImplementedError, ValueError):
```

Comment:
Why `ValueError`?

Reply:
Sorry about that. That was there when the `.entropy` in `ExponentialFamily` returned a `ValueError`. I have removed it now. Thank you.